src/App.css

```diff
@@ -22,3 +22,16 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+.headerDiv {
```

Comment:
Would you please move the added lines to `index.css` line 236?
https://github.com/ourcodeinc/ActiveDocumentation-webapp/blob/main/src/index.css#L236